src/shared/ui/inputs/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,50 @@
+import { VariantProps, cva } from "cva";
+import { Input } from "../Input/Input";
+
+export const searchInputVariants = cva({
+  base: "block w-full border-0 bg-background-tinted-base pl-spacing-looser-3 shadow-essential-base placeholder:text-font-size-smaller",
+
+  variants: {
+    rounded: {
+      full: "rounded-full border-border-width-hairline border-transparent py-spacing-tighter placeholder:text-essential-subdued hover:border-essential-subdued hover:bg-background-tinted-highlight focus:border-border-width-hairline focus:border-essential-base focus:border-white focus:ring-white",
+      square:
+        "rounded-border-radius-base py-spacing-tighter-5 placeholder:text-color-subdued focus:ring-transparent",
+    },
+  },
+});
```

Comment:
Great to have used cva ✌️ !

If I may, I would split the `rounded` variant with at least a `size` one:

- The `rounded` variant is responsible for the radius shape (`full` or `square` eg: `rounded-full` vs `rounded-border-radius-base`)
- The `size` variant would be responsible of… the size of the input (eg: `py-spacing-tighter` vs py-spacing-tighter-5).

This way, we could have large square button (or small) and large rounded button (or small).

What do you think?

Reply:
Good point ! I added the `size` variant